continuous_integration/test_script.sh

```diff
@@ -7,12 +7,30 @@ if [[ -n "$FLAKE8_VERSION" ]]; then
 fi
 
 if [[ "$SKIP_TESTS" != "true" ]]; then
+    PYTEST_VERSION=$(python -c "import pytest; print(pytest.__version__)")
+
+    if [ ${PYTEST_VERSION:0:1} == "3" ]; then
+        # pytest >= 3.0.0 Produces a very long error log with pytest
+        # warnings for deprecated yield based tests. This is a hacky way
+        # to produce cleaner CI logs by disabling warnings.
+        # I am not adding this to setup.cfg directly as it is not available
+        # for pytest < 2.8.0
+        sed -i '/addopts/a\    --disable-pytest-warnings' setup.cfg
```

Comment:
I did not have it earlier and `pytest` version was 2.7.2 for python 2.7 - a few jobs gave a red hence I added it to make sure nothing breaks.